sdk/core/core-lro/src/http/operation.ts

```diff
@@ -138,27 +138,21 @@ function transformStatus(inputs: { status: unknown; statusCode: number }): Opera
       `Polling was unsuccessful. Expected status to have a string value or no value but it has instead: ${status}. This doesn't necessarily indicate the operation has failed. Check your Azure subscription or resource status for more information.`,
     );
   }
-  switch (status?.toLocaleLowerCase()) {
-    case undefined:
-      return toOperationStatus(statusCode);
-    case "succeeded":
-      return "succeeded";
-    case "failed":
-      return "failed";
-    case "running":
-    case "accepted":
-    case "started":
-    case "canceling":
-    case "cancelling":
-      return "running";
-    case "canceled":
-    case "cancelled":
-      return "canceled";
-    default: {
-      logger.verbose(`LRO: unrecognized operation status: ${status}`);
-      return status as OperationStatus;
-    }
+  logger.verbose(`Transforming status: ${status} with status code: ${statusCode}.`);
```

Comment:
nit: maybe prefix it with `LRO:` if that's what we do otherwise in this package (I just saw it in the previous diff)